src/useTranslation.js

```diff
@@ -68,12 +70,20 @@ export function useTranslation(ns, props = {}) {
     // unbinding on unmount
     return () => {
       isMounted.current = false;
-      if (bindI18n && i18n) bindI18n.split(' ').forEach(e => i18n.off(e, boundReset));
+      if (bindI18n && i18n) bindI18n.split(' ').forEach((e) => i18n.off(e, boundReset));
       if (bindI18nStore && i18n)
-        bindI18nStore.split(' ').forEach(e => i18n.store.off(e, boundReset));
+        bindI18nStore.split(' ').forEach((e) => i18n.store.off(e, boundReset));
     };
   }, [namespaces.join()]); // re-run effect whenever list of namespaces changes
 
+  const isInitial = useRef(true);
```

Comment:
Can you maybe add an inline comment to explain why this is needed?
For future generations ;-)

Reply:
Done :-)

Reply:
neet -> need

Reply:
Uups. Fixed.